Javadoc generation fails

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:javadoc (default-cli) on project smallrye-health-api: An error has occurred in Javadoc report generation: 
[ERROR] Exit code: 1 - /home/mstefank/GIT/xstefank/smallrye-health/api/src/main/java/io/smallrye/health/api/AsyncHealthCheck.java:25: error: type arguments not allowed here
[ERROR]      * @return {@link Uni<HealthCheckResponse>} object containing information about the health check result
[ERROR]                       ^
[ERROR] /home/mstefank/GIT/xstefank/smallrye-health/api/src/main/java/io/smallrye/health/api/AsyncHealthCheck.java:27: warning - Tag @link:illegal character: "60" in "Uni<HealthCheckResponse>"
[ERROR] /home/mstefank/GIT/xstefank/smallrye-health/api/src/main/java/io/smallrye/health/api/AsyncHealthCheck.java:27: warning - Tag @link:illegal character: "62" in "Uni<HealthCheckResponse>"
[ERROR] /home/mstefank/GIT/xstefank/smallrye-health/api/src/main/java/io/smallrye/health/api/HealthRegistry.java:40: error: tag not allowed here: <pre>
[ERROR]  * <pre>
[ERROR]    ^
[ERROR] /home/mstefank/GIT/xstefank/smallrye-health/api/src/main/java/io/smallrye/health/api/HealthRegistry.java:48: error: tag not allowed here: <pre>
[ERROR]  * <pre>
[ERROR]    ^
```

--- a/api/src/main/java/io/smallrye/health/api/AsyncHealthCheck.java
+++ b/api/src/main/java/io/smallrye/health/api/AsyncHealthCheck.java
@@ -22,7 +22,7 @@ public interface AsyncHealthCheck {
      * check failed. In addition, returning a {@code Uni} propagating a {@code null} value as the item
      * is considered also as a failure.
      *
-     * @return {@link Uni<HealthCheckResponse>} object containing information about the health check result
+     * @return {@link Uni} object containing information about the health check result
      */
     Uni<HealthCheckResponse> call();
 }

--- a/api/src/main/java/io/smallrye/health/api/HealthRegistry.java
+++ b/api/src/main/java/io/smallrye/health/api/HealthRegistry.java
@@ -35,7 +35,8 @@ import io.smallrye.common.annotation.Experimental;
  * </p>
  * 
  * <ul>
- * <li><b>Liveness</b> (also default when no qualifier is defined):</li>
+ * <li><b>Liveness</b></li>
+ * </ul>
  * 
  * <pre>
  * &#64;Inject
@@ -43,14 +44,15 @@ import io.smallrye.common.annotation.Experimental;
  * HealthRegistry livenessHealthRegistry;
  * </pre>
  * 
+ * <ul>
  * <li><b>Readiness</b>:</li>
+ * </ul>
  * 
  * <pre>
  * &#64;Inject
  * &#64;Readiness
  * HealthRegistry readinessHealthRegistry;
  * </pre>
- * </ul>
  */
 @Experimental("Programmatic Health API")
 public interface HealthRegistry {

